docs/source/index.rst

```diff
@@ -127,21 +127,22 @@ Welcome to AiiDA's documentation!
    :caption: Reference
    :hidden:
 
-   reference/placeholder
+   reference/command_line
+   apidoc/aiida.rst
+   reference/rest_api
 
 .. toctree::
    :maxdepth: 2
-   :caption: Plugins
-   :hidden:
-
-   plugins/placeholder
+   :caption: Core Development
```

Comment:
Same comment as above, I believe this section should be called "Internal architecture". Also, headers should be in sentence case, see the wiki style guide.

Reply:
Sounds good to me, I'll think a bit about it.

Reply:
E.g. "documentation cookbook" would then not really fit...

Reply:
I agree, that might need to go into the wiki then. But we can also just live with the slight inconsistency and fix that at some point later.